### PR TITLE
Clarify the env requirements in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ansman**
The wording is a bit confusing and does not clearly indicate that you can set `JDK_XX` to whatever JDK version you want.

### CODE REVIEW

**Review by LouisCAD — APPROVED** [review]
Great addition, will lower the entry barrier to run the tests for external contributors before committing and submitting their work.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -79,9 +79,16 @@ to Gradle (in Preferences -> Build, Execution, Deployment -> Build Tools -> Grad
 
 * JDK >= 11 referred to by the `JAVA_HOME` environment variable.
 * JDK 1.6 referred to by the `JDK_16` environment variable. 
-  It is OK to have `JDK_16` pointing to `JAVA_HOME` for external contributions.
+  It is OK to have `JDK_16` pointing to a non 1.6 JDK (e.g. `JAVA_HOME`) for external contributions.
 * JDK 1.8 referred to by the `JDK_18` environment variable. Only used by nightly stress-tests. 
-  It is OK to have `JDK_18` pointing to `JAVA_HOME` for external contributions.
+  It is OK to have `JDK_18` to a non 1.6 JDK (e.g. `JAVA_HOME`) for external contributions.
```
Comment (EdwarDDay) [inline CONTRIBUTING.md:84]: This looks like a copy paste mistake.
```suggestion
  It is OK to have `JDK_18` to a non 8 JDK (e.g. `JAVA_HOME`) for external contributions.
```

**Review by elizarov — APPROVED** [review]
A helpful clarification. Thanks!

**Comment by ansman** [comment]
@elizarov Thanks for fixing the typo

**Comment by LouisCAD** [comment]
Having "1.6" and "8" just below is a little inconsistent, don't you think?

**Comment by elizarov** [comment]
> Having "1.6" and "8" just below is a little inconsistent, don't you think?

Fixed: https://github.com/Kotlin/kotlinx.coroutines/commit/2554f560d357a7161a4f79fc0d1ea9f3bf2cfb5b

**Pull request merged**